proto/rill/admin/v1/api.proto

```diff
@@ -971,6 +971,8 @@ message GetProjectRequest {
   uint32 access_token_ttl_seconds = 3;
   bool superuser_force_access = 5;
   bool issue_superuser_token = 4;
+  // Skips fetching deployment and jwt related to querying deployment
+  bool skip_deployment = 6;
```

Comment:
Are you going to be calling this endpoint in a poll loop? Otherwise, I'd prefer to just keep the API simple and always return the deployment info (it's relatively cheap to generate).